### ISSUE TITLE
CoP Templates and Guides Board Migration

### ISSUE THREAD

**Issue by edwardsarah** [hackforla/guides#169]
### Overview
GitHub is retiring classic project boards. We need to migrate all guides to the new Guides Tracker from the old project boards. 

### Action Items
- [ ] Engineering
- [ ] Design Systems
- [ ] Data Science
- [ ] Marketing
- [ ] DevOps
- [ ] Product
- [ ] Revenue/Fundraising
- [ ] UI/UX
- [ ] Admin

### Resources/Instructions
- [Guides Tracker Completeness Instructions](https://github.com/hackforla/guides/wiki/Guides-Tracker-Completeness-Instructions)
- [Classic Projects Guides Tracker](https://github.com/orgs/hackforla/projects/3)
- [New Guides Tracker](https://github.com/orgs/hackforla/projects/58/views/1)

**Instructions to Copy** 
```
[COP Name]
- [ ] Check if the CoP has already created a new board. If so, link here: [INSERT LINK]
- [ ] Check the Guides Tracker Completeness for any guide issues not on the classic project board
- [ ] Add any missing issues to the classic project board
- [ ] Add the label for the COP to all issues
- [ ] Add all issues to the new Guides Tracker
- [ ] Check that they have been added correctly
- [ ] Create a new view on the Guides Tracker for that CoP
    - [ ] Duplicate the existing view
    - [ ] Rename the view
    - [ ] Filter by the CoP
```

**Comment by edwardsarah** [comment]
## Engineering. Templates and Guides Board
[Engineering Templates and Guides - New Guides Tracker](https://github.com/orgs/hackforla/projects/58/views/5)

### Checklist
- [x] [Engineering Guides & Templates Board](https://github.com/hackforla/engineering/projects/4)
- [x] Check the Guides Tracker Completeness for any guide issues not on the classic project board
- [x] Add any missing issues to the classic project board
- [x] Add the label for the COP to all issues
- [x] Add all issues to the new Guides Tracker
- [x] Check that they have been added correctly
- [x] Create a new view on the Guides Tracker for that CoP
    - [x] Duplicate the existing view
    - [x] Rename the view
    - [x] Filter by the CoP

### Project Board Card Text
#### Resources
- [x] needed
- [ ] not needed
```
[Career development roadmaps](https://roadmap.sh/roadmaps)
```

- [x] needed
- [ ] not needed
```
Welcome! 

We are creating guides that will help future HfLA volunteers at Hack for LA, and add to valuable shared documentation between all teams.

[Resources](https://github.com/hackforla/guides/wiki/Guide-Resources)  to help you navigate making, contributing and using Guides 
```

- [ ] needed
- [x] not needed
```
**Hack for LA png**

![Hack for LA png](https://user-images.githubusercontent.com/37763229/103038336-0a112600-4523-11eb-975e-25b942f7bc2e.png)
```

- [ ] needed
- [x] not needed
```
[How to add screenshot & graphics to your guide](https://drive.google.com/file/d/1liK5u5_jKa06h3pFng4O7zC9pcf_jJsN/view)
```

- [x] needed
- [ ] not needed
```
[Software Development Lifecycle](https://drive.google.com/file/d/1uZ5QG0N4--nQ-jMy1BfKuZSe7Wumhtb3/view) encompasses all steps needed to take a product from concept to market availability. 
```

- [ ] needed
- [x] not needed
```
[Design Sprint Templates](https://designsprintkit.withgoogle.com/resources/overview) that were recommended by Maicol of Super by Design
```

- [x] needed
- [ ] not needed
```
Use this [Full Page Screen Capture Chrome Extension](https://chrome.google.com/webstore/detail/gofullpage-full-page-scre/fdpohaocaechififmbbbbbknoalclacl?hl=en) capture a screenshot of your current page in entirety and reliably.
```

- [ ] needed
- [x] not needed
```
Use [Docs to Markdown Chrome Extension](https://workspace.google.com/marketplace/app/docs_to_markdown/700168918607) to convert a Google Doc to simple, readable Markdown or HTML.
```

- [x] needed
- [ ] not needed
```
Use this [Color Picker Chrome Extension](https://chrome.google.com/webstore/detail/eye-dropper/hmdcmlfkchdmnmnmheododdhjedfccka/related) to get the right colors for text in One Sheets and on other documents.
```

---
All links have been copied and the wiki for engineering reviewed

**Comment by edwardsarah** [comment]
## Ops
[Ops Templates and Guides - New Guides Tracker](https://github.com/orgs/hackforla/projects/58/views/11)

### Checklist
- [x] [Current Guides Tracker - Ops](https://github.com/hackforla/ops/projects/2)
- [x] Check the Guides Tracker Completeness for any guide issues not on the classic project board
- [x] Add any missing issues to the classic project board
- [x] Add the label for the COP to all issues
- [x] Add all issues to the new Guides Tracker
- [x] Check that they have been added correctly
- [x] Create a new view on the Guides Tracker for that CoP
    - [x] Duplicate the existing view
    - [x] Rename the view
    - [x] Filter by the CoP

### Project Board Card Text
#### Resources
```
Welcome! 

We are creating guides that will help future HfLA volunteers at Hack for LA, and add to valuable shared documentation between all teams.

[Resources](https://github.com/hackforla/guides/wiki/Guide-Resources)  to help you navigate making, contributing and using Guides 
```
#### New Issue Approval
```
Future Guides
- Adding a project to incubator [#20](https://github.com/hackforla/ops/issues/20)
- How to access logs to debug with your team
- What to store in 1password (Secrets, ports tokens)
- When to graduate your project from incubator
- Setting up Dev & Production Environments 
   - AWS
   - Azure
   - gh-pages
- How to get added as a credentialed user on AWS
- Using Terraform
- Using Terragrunt (required for incubator)
- Setup Docker
```

**Comment by edwardsarah** [comment]
Ops board has some issues labeled feature: guide creation which do not appear to be guides. These have been omitted from the classic guides tracker.

**Comment by edwardsarah** [comment]
## Admin
[Admin Templates and Guides - New Guides Tracker](https://github.com/orgs/hackforla/projects/58/views/6)

### Checklist

- [x] [Old Admin Templates and Guides Board](https://github.com/hackforla/admin/projects/1)
- [x] Check the Guides Tracker Completeness for any guide issues not on the classic project board
- [x] Add any missing issues to the classic project board
- [x] Add the label for the COP to all issues
- [x] Add all issues to the new Guides Tracker
- [x] Check that they have been added correctly
- [x] Create a new view on the Guides Tracker for that CoP
    - [x] Duplicate the existing view
    - [x] Rename the view
    - [x] Filter by the CoP

### Project Board Card Text

#### Start Here/Links

- Start Here = Includes things like meeting times and zoom links, and anything the team members need to read.
  - [X] needed  
  - [ ] not needed

- Welcome!  
  We are creating guides that will help future HfLA volunteers at Hack for LA, and add to valuable shared documentation between all teams.  
  [Resources](https://github.com/hackforla/guides/wiki/Guide-Resources) to help you navigate making, contributing and using Guides  
  - [x] needed  
  - [ ] not needed

- Team meets  
  - Sundays 2pm PDT  
  - Adhoc  
  - [x] needed  
  - [ ] not needed

- Links:  
  - Slack channel [#admin-team](https://hackforla.slack.com/archives/C027HDTJJ94)  
  - [Default Project Board Template](https://github.com/hackforla/.github/projects/2)  
  - [Admin Google Drive - new](https://drive.google.com/drive/folders/1RC5Hx_nGhrYUNm8O-u1gbwC9bLrBy9I7?usp=sharing)  
  - [Admin Google Drive - old](https://drive.google.com/drive/folders/16O2Ubj-gIGrXXJ2pTt4cVlPycmwQnKLi)  
  - [Admin FAQ](https://docs.google.com/document/d/1vgLsPrGY_P6UlJeaoBw47Om2drmzqH0BT1XmRruIcUk/edit)  
  - [Admin Team Roster](https://docs.google.com/spreadsheets/d/1wHXgc3ApcFGX_DGpur0pj4d3ocfPmfVilhr-Eop8ZaY/edit#gid=0)  
  - [Recordings](https://drive.google.com/drive/folders/1i_mjlTp0tNfUiXeXN4YmbbuJg6iF-hPz)  
  - [GitHub Repository Permission Levels Matrix](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization)  
  - [x] needed  
  - [ ] not needed

- Platforms we use (if you think of more please add):  
  - [x] needed  
  - [ ] not needed  
  - Slack https://github.com/hackforla/admin/issues/2  
  - Meetup https://github.com/hackforla/admin/issues/3  
  - GitHub https://github.com/hackforla/admin/issues/4  
  - Gsuite https://github.com/hackforla/admin/issues/5  
  - 1Password https://github.com/hackforla/admin/issues/6  
  - Google Drive https://github.com/hackforla/admin/issues/7  
  - Miro https://github.com/hackforla/admin/issues/11  
  - Figma https://github.com/hackforla/admin/issues/12  

#### Ice Box

- Icebox = issues identified that are going to get addressed, but are not being started yet because of other more pressing issues. Or issues that are not fully developed.  
  - [x] needed  
  - [ ] not needed

- Admin calendar that shows deadlines/renewals, etc for the organization - KC  
  - [x] needed  
  - [ ] not needed

#### New Issue Approval

- Bonnie will create a template issue in Marketing Repo for the social media logins in 1Password  
- 1Password, make sure there is a delete vault guide  
  - [x] needed  
  - [ ] not needed

#### Prioritized Backlog

- Prioritized Backlog = features that are ready to work and a priority. This column should be prioritized by most important first.  
  - [x] needed  
  - [ ] not needed

#### In Progress

- In Progress = tasks that are actively being worked on by an individual team member and have been assigned to them in the issue.  
  - [x] needed  
  - [ ] not needed

#### Questions & Review

- Links/Questions/Review = Cards that have links that you need to get to often. Issues that you are blocked on because you need an answer from another team member or other type of question/review.  
  - [x] needed  
  - [ ] not needed

#### Guidance for font decoration  
- Bold or Strong — Text or buttons that should be clicked/selected (e.g., Sign-In): **Confirm**  
- Quotes — Text that the user enters word-for-word (e.g., user is instructed to enter xyz into this field): Enter `/Remind`  
- Italics — Text that refers to something on the page: Their name and email will appear in the _Waiting to be Confirmed_ header.  
  - [x] needed  
  - [ ] not needed

**Comment by edwardsarah** [comment]
## Revenue
[Revenue Templates and Guides - New Guides Tracker](https://github.com/orgs/hackforla/projects/58/views/7)

### Checklist
- [x] Check if the CoP has already created a new board. If so, link here: 
    - Revenue does not have a dedicated Templates and Guides board 
    - [Issues labeled feature: guide creation](https://github.com/hackforla/revenue/issues?q=is%3Aissue+is%3Aopen+label%3A%22feature%3A+guide+creation%22)
- [x] Check the Guides Tracker Completeness for any guide issues not on the classic project board
- [x] Add any missing issues to the classic project board
- [x] Add the label for the COP to all issues
- [x] Add all issues to the new Guides Tracker
- [x] Check that they have been added correctly
- [x] Create a new view on the Guides Tracker for that CoP
    - [x] Duplicate the existing view
    - [x] Rename the view
    - [x] Filter by the CoP

### Project Board Card Text
None

**Comment by edwardsarah** [comment]
## Marketing
[Marketing Templates and Guides - New Guides Tracker](https://github.com/orgs/hackforla/projects/58/views/8)

### Checklist
- [x] Check if the CoP has already created a new board. 
    - Marketing does not have a dedicated board for Guides and Templates 
    - [Issues labeled feature: guide creation](https://github.com/hackforla/marketing/labels/feature%3A%20guide%20creation)
- [x] Check the Guides Tracker Completeness for any guide issues not on the classic project board
- [x] Add any missing issues to the classic project board
- [x] Add the label for the COP to all issues
- [x] Add all issues to the new Guides Tracker
- [x] Check that they have been added correctly
- [x] Create a new view on the Guides Tracker for that CoP
    - [x] Duplicate the existing view
    - [x] Rename the view
    - [x] Filter by the CoP

### Project Board Card Text
None

**Comment by edwardsarah** [comment]
## Data Science
[Data Science Templates and Guides - New Guides Tracker](https://github.com/orgs/hackforla/projects/58/views/9)

### Checklist
- [x] Check if the CoP has already created a new board. If so, link here:
    - Data science does not have a designated templates and guides board.  
- [x] Check the Guides Tracker Completeness for any guide issues not on the classic project board
- [x] Add any missing issues to the classic project board
- [x] Add the label for the COP to all issues
- [x] Add all issues to the new Guides Tracker
- [x] Check that they have been added correctly
- [x] Create a new view on the Guides Tracker for that CoP
    - [x] Duplicate the existing view
    - [x] Rename the view
    - [x] Filter by the CoP

### Project Board Card Text
None

**Comment by edwardsarah** [comment]
## Product
[Product Templates and Guides - New Guides Tracker](https://github.com/orgs/hackforla/projects/58/views/10)

### Checklist
- [x] [Old Product Templates and Guides Board](https://github.com/hackforla/product-management/projects/2)
- [x] Check the Guides Tracker Completeness for any guide issues not on the classic project board
- [x] Add any missing issues to the classic project board
- [x] Add the label for the COP to all issues
- [x] Add all issues to the new Guides Tracker
- [x] Check that they have been added correctly
- [x] Create a new view on the Guides Tracker for that CoP
    - [x] Duplicate the existing view
    - [x] Rename the view
    - [x] Filter by the CoP

### Project Board Card Text
#### Resources

- [x] **Needed**  
- [x] **Not needed**  
Welcome!  
We are creating guides that will help future HfLA volunteers at Hack for LA, and add to valuable shared documentation between all teams.  
[Resources](https://github.com/hackforla/guides/wiki/Guide-Resources) to help you navigate making, contributing and using Guides

- [x] **Needed**  
- [ ] **Not needed**  
Use this [Color Picker Chrome Extension](https://chrome.google.com/webstore/detail/eye-dropper/hmdcmlfkchdmnmnmheododdhjedfccka/related) to get the right colors for text in One Sheets and on other documents.

- [x] **Needed**  
- [ ] **Not needed**  
**Hack for LA logo png**  
![HfLA logo](https://user-images.githubusercontent.com/37763229/103038336-0a112600-4523-11eb-975e-25b942f7bc2e.png)

- [x] **Needed**  
- [ ] **Not needed**  
[Design Sprint Templates](https://designsprintkit.withgoogle.com/resources/overview) that were recommended by Maicol of Super by Design

#### New Issue Approval

- [x] **Needed**  
- [ ] **Not needed**  
Showcase Deck  
Marketing Board [Issue](https://github.com/hackforla/marketing/projects/1#card-51043445)  
_this goes to another card on another board_

**Project Showcase Deck examples:**
- [TDM](https://docs.google.com/presentation/d/1djX48CJLJtw0ObDLN9cpVhJEIVsN4dplc4pe_wciUQY/edit#slide=id.p1)
- [Civic Tech Index](https://docs.google.com/presentation/d/1dH2Fdo6ScN-4TRcbVseP3bIMxKU-sI3gmRRy7S1X3d4/edit#slide=id.g9cf4e56a7a_1_5)

**Needed:**
- Guide to what a showcase deck should contain

- [x] **Needed**  
- [ ] **Not needed**  
Fonts - Google Fonts

- [x] **Needed**  
- [ ] **Not needed**  
**Repository Documentation: SOP Explanation**  
`README.md` should include an overview, goals, development processes, technology, etc.  
The repository's wiki should capture institutional knowledge and guides for new contributors.

- [x] **Needed**  
- [ ] **Not needed**  
**Using Images on HfLA projects**  
[Guide Issue](https://github.com/hackforla/knowledgebase-content/issues/105)

**Comment by edwardsarah** [comment]
Note: we did UI/UX together during the last meeting, but its project board cards will still need to be copied to preserve them!

**Comment by JesseTheCleric** [comment]
## UI/UX Templates and Guides Board
[UI/UX Templates and Guides - New Guides Tracker](https://github.com/orgs/hackforla/projects/58/views/4)
### Checklist
- [ ] [Ui/UX Templates and Guides](https://github.com/hackforla/UI-UX/projects/1)
- [ ] Check the Guides Tracker Completeness for any guide issues not on the classic project board
- [ ] Add any missing issues to the classic project board
- [ ] Add the label for the COP to all issues
- [ ] Add all issues to the new Guides Tracker
- [ ] Check that they have been added correctly
- [ ] Create a new view on the Guides Tracker for that CoP
    - [ ] Duplicate the existing view
    - [ ] Rename the view
    - [ ] Filter by the CoP

### Project Board Card Text
#### Instructions

- [x] **Needed** 
- [ ] **Not needed**   
### Welcome!  
We are creating guides that will help future HfLA volunteers at Hack for LA, and add to valuable shared documentation between all teams.

#### Resources

- [x] **Needed** 
- [ ] **Not needed**  
[Resources](https://github.com/hackforla/guides/wiki/Guide-Resources) to help you navigate making, contributing and using Guides

#### Guidelines

- [x] **Needed** 
- [ ] **Not needed**  
- Volunteers with over 3 weeks of inactivity are unassigned from their "guides" issue  
- Unassigned guide is moved back to prioritized backlog  

#### Tracker

- [x] **Needed** 
- [ ] **Not needed**  
- Tracker: Templates & Guides [#15](https://github.com/hackforla/knowledgebase-content/issues/97)

#### Backlog

- [x] **Needed** 
- [ ] **Not needed**  
Learn Hack for LA projects’ effective processes and practices by working on a guide issue! 💪

**Comment by ExperimentsInHonesty** [comment]
## Guides Tracker old board

### Needs to be Triaged

- [ ] This one got edited down and added as header on the new board
```
Welcome to the Hack for LA Guide Tracker! 

We are creating guides that will help current and future HfLA volunteers at Hack for LA, and add to valuable shared documentation between all teams.

Search for a guide by using this global project board. The Guides team uses this board to track progress of all the guides

See the [Guides Team WIKI](https://github.com/hackforla/guides/wiki/Guide-Resources) for details about:
- Where to start
- How to suggest a new guide
- Once you are ready to write a guide
- Tools to help you connect with projects
```

- [ ] no longer on the new board
```
Links for checking that all guides are listed on this board
- [Guides Tracker Completeness Instructions](https://github.com/hackforla/guides/wiki/Guides-Tracker-Completeness-Instructions)

```

**Comment by fchan218** [comment]
we need labels for: 
- CoP label: check needed
- CoP label: checked
- CoP folder shortcut needed

**Comment by ExperimentsInHonesty** [comment]
Things found that might go other places

```
[Design Sprint Templates](https://designsprintkit.withgoogle.com/resources/overview) that were recommended by Maicol of Super by Design
```
